types/stylelint/index.d.ts

```diff
@@ -180,7 +182,7 @@ declare module 'stylelint' {
 		optional?: boolean;
 	};
 
-	type RuleContext = {
+	export type RuleContext = {
```

Comment:
[suggest] Also here:

```suggestion
	/**
	 * A rule context.
	 */
	export type RuleContext = {
```